openkfw/TruBudget
# Close (sub)project warning message

### Checklist




### Description


### How to test

1. Create new project and subproject
2. Close subproject
3. Close project
4. In both cases you should see the warning in all languages




Closes #1871
